stories/molecules/UkraineBannerExample.tsx

```diff
@@ -5,9 +5,9 @@ const UkraineBannerExample = () => {
   return (
     <BannerCard
       link="/"
-      title="Lær om det norske samfunn - på ukrainsk"
-      content="Дізнайтеся про норвезьке суспільство – українською"
-      linkText="Learn about Norwegian society - in Ukrainian"
+      title={{ title: 'Lær om det norske samfunn - på ukrainsk', lang: 'nb' }}
+      content={{ content: 'Дізнайтеся про норвезьке суспільство – українською', lang: 'uk' }}
```

Comment:
Vi har brukt ukr i ed.
```suggestion
      content={{ content: 'Дізнайтеся про норвезьке суспільство – українською', lang: 'ukr' }}
```

Reply:
Men er det en gyldig language-verdi i HTML-specet?

Reply:
Hmm. Ser at vi burde ha brukt uk som kode i ed også.

Reply:
Lurer på om vi skal bytte til uk i backend.